tests/core/test_settings.py

```diff
@@ -31,7 +31,11 @@
 from airflow.api_internal.internal_api_call import InternalApiConfig
 from airflow.configuration import conf
 from airflow.exceptions import AirflowClusterPolicyViolation, AirflowConfigException
-from airflow.settings import _ENABLE_AIP_44, TracebackSession, is_usage_data_collection_enabled
+from airflow.settings import (
+    _ENABLE_AIP_44,
+    TracebackSession,
+    is_usage_data_collection_enabled,
+)
```

Comment:
Nit: unrelated changes in this file now, lets revert these ones

```suggestion
from airflow.settings import _ENABLE_AIP_44, TracebackSession, is_usage_data_collection_enabled
```